biome.json

```diff
@@ -9,30 +9,219 @@
     "lineWidth": 80,
     "attributePosition": "auto"
   },
-  "organizeImports": { "enabled": true },
+  "assist": {
+    "actions": {
+      "source": {
+        "organizeImports": "on"
+      }
+    }
+  },
   "linter": {
     "enabled": true,
     "rules": {
-      "all": true,
       "a11y": {
+        "noAccessKey": "error",
+        "noAriaHiddenOnFocusable": "error",
+        "noAriaUnsupportedElements": "error",
+        "noBlankTarget": "error",
```

Comment:
This seems to prevent using the command `biome lint`.
```
biome.json:26:9 deserialize ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Found an unknown key `noBlankTarget`.
  
    24 │         "noAriaHiddenOnFocusable": "error",
    25 │         "noAriaUnsupportedElements": "error",
  > 26 │         "noBlankTarget": "error",
       │         ^^^^^^^^^^^^^^^
```

Reply:
This was a change from 2.0.0-beta1 to 2.0.0-beta4 - need to lock your Biome version!